tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -184,6 +199,14 @@ partial class Build
                 source = sourceDir / "Datadog.Profiler.Native.pdb";
                 dest = SymbolsDirectory / $"win-{architecture}" / Path.GetFileName(source);
                 CopyFile(source, dest, FileExistsPolicy.Overwrite);
+
+
+                source = ProfilerOutputDirectory / "bin" / $"{BuildConfiguration}-{architecture}" / "profiler" / "test" / "Datadog.Profiler.Native.Tests" / "Datadog.Profiler.Native.Tests.exe";
+
+                dest = SharedDirectory / "bin" / "test" / $"win-{architecture}";
+                EnsureExistingDirectory(dest);
+
+                CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
```

Comment:
As above, I don't think this should be in the "Publish" stage personally, but maybe it's not worth worrying about for now